Toshi/Source/Toshi/Strings/TCString.h

```diff
@@ -38,8 +40,8 @@ namespace Toshi
 		int Compare(const char*, int) const;
 		int CompareNoCase(const char*, int) const;
 
-		inline TCString& MakeUpper() { if (m_pBuffer != TNULL) { strupr(m_pBuffer); } return *this; }
```

Comment:
m_pBuffer cannot be equal to TNULL, it can only be equal to NullString which is a pointer to few null bytes so the check is useless

Reply:
Yeah, idk why they made it like that, but you're right